test/commands/UserInputUtil.test.ts

```diff
@@ -898,6 +898,48 @@ describe('UserInputUtil', () => {
                 placeHolder: 'choose a folder'
             });
         });
+
+        it('should show the workspace folders along with their path as a description', async () => {
+            mySandBox.stub(UserInputUtil, 'getWorkspaceFolders').returns([
+                {
+                    name: 'myPath1',
+                    uri: {
+                        path: 'path1'
+                    }
+                },
+                {
+                    name: 'myPath2',
+                    uri: {
+                        path: 'path2'
+                    }
+                }]);
+
+            quickPickStub.resolves({
+                label: 'myPath1', data: {
+                    name: 'myPath1',
+                    uri: {
+                        path: 'path1'
+                    }
+                }, description : 'pathHere'
+            });
+
+            const result: IBlockchainQuickPickItem<vscode.WorkspaceFolder> = await UserInputUtil.showWorkspaceQuickPickBox('choose a folder');
+            result.should.deep.equal({
+                label: 'myPath1', data: {
+                    name: 'myPath1',
+                    uri: {
+                        path: 'path1'
+                    }
+                }, description : 'pathHere'
+            });
+
+            quickPickStub.should.have.been.calledWith(sinon.match.any, {
```

Comment:
you should change `sinon.match.any` to be the quick pick items that should be shown so you can test the it shows the correct description

Reply:
I believe I have now got the correct quick pick items - test passes.